eng/tools/spec-gen-sdk-runner/src/commands.ts

```diff
@@ -130,6 +131,19 @@ export async function generateSdkForSpecPr(): Promise<number> {
       // Read the execution report to aggreate the generation results
       executionReport = getExecutionReport(commandInput);
       currentExecutionResult = executionReport.executionResult;
+      if (executionReport.stagedArtifactsFolder) {
+        commandInput.stagedArtifactsFolder = executionReport.stagedArtifactsFolder;
+      }
+      
+      for (const pkg of executionReport.packages) {
+        if (serviceNames.length > 0) {
+          serviceNames += ",";
+        }
+        if (pkg.serviceName) {
```

Comment:
when serviceName is empty, extra ',' will be added to serviceNames. We might move line#140 to within 'if' block then remove the ending ',' at last.